app/assets/javascripts/application.js

```diff
@@ -14,3 +14,9 @@
 //= require activestorage
 //= require turbolinks
 //= require_tree .
+
+function stuff() {
+  var d = '11';
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
'd' is assigned a value but never used       no-unused-vars